.erb/scripts/windows-sign.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+// Custom Sign hook for use with electron-builder + SSL CodeSignTool
+
+const { execSync } = require("child_process");
+
+exports.default = async (config) => {
+  const ES_CREDENTIAL_ID = process.env.ES_CREDENTIAL_ID;
+  const ES_USERNAME = process.env.ES_USERNAME;
+  const ES_PASSWORD = process.env.ES_PASSWORD;
+  const ES_TOTP_SECRET = process.env.ES_TOTP_SECRET;
+  const CODESIGNTOOL_PATH = process.env.CODESIGNTOOL_PATH;
```

Comment:
Do you mind also moving these outside/above this function, so they appear more like actual constants?